### ISSUE TITLE
Changing a tag

### ISSUE THREAD

**Issue by philiprbrenan** [geany/geany#664]
![changetag](https://cloud.githubusercontent.com/assets/7548378/10077386/fe79fb74-62d7-11e5-9d92-9bfca9360872.png)

If one adds a new tag, <b>Geany</b> (very helpfully) adds the closing tag - line 22.  But if one changes a tag that already has a matching closing tag it does not change the closing tag to match - line 23.  Should it be changed to make this behaviour more consistent?

**Comment by frlan** [comment]
Indeed. I recognized this earlier too. But we have to be careful, as it will might end in unexpected behaviour -- the user might not expect at all, that the closing tag will be touched.

**Comment by elextr** [comment]
Agree with @frlan that it is much harder to decide what is the "right" thing to do when editing existing elements instead of adding new elements where its reasonably certain you need a closing tag.
